<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectUnityWindow</class>
 <widget class="QWidget" name="ConnectUnityWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ConnectUnityWindow</string>
  </property>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Connect Unity</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>82</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Connect">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>340</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_start">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>340</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_stop">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>340</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>691</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
